package-overrides/github/sytelus/cryptojs@3.1.2.json

```diff
@@ -0,0 +1,129 @@
+{
+  "main": "core.js",
+  "format": "global",
+  "directories": {
+    "lib": "components"
+  },
+  "shim": {
+    "core.js": {
+      "exports": "CryptoJS"
+    },
+    "sha224.js": {
+      "deps": [
+        "cryptojs/sha256.js"
+      ]
```

Comment:
I'm wondering if it's worth setting exports here as well - `"exports": "CryptoJS.algo.SHA224"` sort of thing?